deploy/backplane/addon-connectors/addon-connectors/10-addon-connectors-admins.SubjectPermission.yml

```diff
@@ -7,6 +7,6 @@ spec:
   permissions:
   - allowFirst: true
     clusterRoleName: admin
-    namespacesAllowedRegex: (^redhat-openshift-connectors$)
+    namespacesAllowedRegex: (^redhat-openshift-connectors.*$)
```

Comment:
Can you give examples of the additional namespaces that would be provided here?

Reply:
Something like `redhat-openshift-connectors-cc6ae6o7764p8lrcfbk0`

Reply:
Oh, these namespaces are randomly generated somehow?

Reply:
Yes exactly.